### ISSUE TITLE
Product Reviews causes horizontal scrolling

### ISSUE THREAD

**Issue by ravenokko** [shopware/shopware#20380]
Product reviews (reading/writing) triggers horizontal scrolling which does not meet the success criteria for [SC 1.4.10](https://www.w3.org/WAI/WCAG22/Understanding/reflow)

<img width="25%" height="25%" alt="Image" src="https://github.com/user-attachments/assets/ef4d86c0-f186-4757-b678-156685dc6159" />
<img width="25%" height="25%" alt="Image" src="https://github.com/user-attachments/assets/22bf192d-db75-4b00-b91a-d0e5824dba68" />
<img width="25%" height="25%" alt="Image" src="https://github.com/user-attachments/assets/b4362b8d-4411-4870-96dd-bc3f241969a1" />

### How to reproduce
In Firefox go to Preferences > General > Zoom
- Ensure default Zoom is set to 100%
- check Zoom text only

Create shop with demodata

Visit storefront homepage
- Set viewport to 320px wide
- Zoom text to 200%
- Open product detail > reviews
